docs/adrs/0012-exposure-events-and-coenrolling-features.md

```diff
@@ -0,0 +1,142 @@
+# Exposure events and coenrolling features
+
+* Status: proposed
+* Deciders: dberry@mozilla.com, jhugman@mozilla.com, jlockhart@mozilla.com, shell@mozilla.com
+* Date: 2023-07-04, making the decision on or before 2023-07-11.
+
+Technical Story: [Supporting exposure events for coenrolled features](https://mozilla-hub.atlassian.net/browse/EXP-3602)
+
+## Context and Problem Statement
+
+Application feature code can be configured from and by Nimbus. In the case of `messaging` and `onboarding`, the complete set of messages and all their associated data is implemented by this feature configuration.
+
```

Comment:
I thought that:
If the default configuration for a feature and no experiment = default config
if a user is enrolled in an experiment - it uses the experiment values + default values for any not defined.  
If a user is enrolled in a rollout - it uses the rollout values + default values for any not defined.
If a user is enrolled in an experiment AND a rollout - it applies all the experiment values + default values for any not applied.